packages/paste-core/components/anchor/src/index.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
-import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+import {safelySpreadTextProps} from '@twilio-paste/text';
+import {DefaultAnchor} from './DefaultAnchor';
+import {InverseAnchor} from './InverseAnchor';
+import {AnchorProps, AnchorPropTypes, AnchorVariants} from './types';
```

Comment:
I think because you've moved them, if anyone is doing

```
import {AnchorProps} from '@twilio-paste/anchor'
```

You may break them. Might need to export the types from here

`export * from './types'` at the bottom probably